enterprise/frontend/src/metabase-enterprise/auth/components/SettingsJWTForm/SettingsJWTForm.tsx

```diff
@@ -16,24 +16,19 @@ import { Flex, Stack } from "metabase/ui";
 import { FormSection } from "metabase/containers/FormikForm";
 import GroupMappingsWidget from "metabase/admin/settings/containers/GroupMappingsWidget";
 import type { SettingValue } from "metabase-types/api";
+import type { SettingElement } from "metabase/admin/settings/types";
 
 type SettingValues = { [key: string]: SettingValue };
 
-type SettingElement = {
-  // Similar to SettingElement from "metabase/admin/settings/types" but with required key
-  key: string;
-  display_name?: string;
-  description?: string;
+type JWTFormSettingElement = Omit<SettingElement, "key"> & {
+  key: string; // ensuring key is required
   is_env_setting?: boolean;
   env_name?: string;
-  placeholder?: string;
   default?: any;
```

Comment:
Should we set default to string here? since all the defaults in this form should be strings? Generally speaking we shouldn't use `any` as a type

Reply:
Fixed here: 621ed0484bd8772f64cd16b350049f684405a75e

Just empirically based on values returned by the getSectionsWithPlugins selector, not all values are strings, and can be other types based on the hard-coded elements I pasted into the unit test.

Reply:
There were more errors that popped up, so I just set it to string as suggested, and made the unit tests work with it.